mep3_simulation/webots_data/controllers/opponent_robot/opponent_robot.py

```diff
@@ -0,0 +1,92 @@
+import random
+
+from controller import Supervisor
+# create the Robot instance.
+
+supervisor = Supervisor()
+
+# get the time step of the current world.
+timestep = int(supervisor.getBasicTimeStep())
+opponent_node = supervisor.getSelf()
+name_field = opponent_node.getField('name')
+opponent_field = opponent_node.getField('translation')
+name = name_field.getSFString()
+
+position = [0, 0, 0]
+
+positions = [(-1.34, 0.437, 10), (-1.12, 0.706, 10), (-0.761, 0.786, 20),
+             (-0.13, 0.792, 10), (-0.764, 0.53, 10), (-0.848, 0.297, 15),
+             (-0.713, 0.063, 10), (-0.603, -0.127, 15)]
+
+positions_1 = [(-1.4, 0.0238, 10), (-1.22, -0.257, 10), (-1.13, -0.646, 10),
+               (-0.738, -0.904, 15), (-0.565, -0.666, 20),
+               (-0.827, -0.412, 15)]
+
+
+def destination_achieved(curr, dest, epsilon):
+    return (abs(curr[0] - dest[0]) < epsilon
+            and abs(curr[1] - dest[1]) < epsilon)
+
+
+def time_achieved(time_period):
+    ret_value = False
+    t = supervisor.getTime()
+
+    while supervisor.getTime() - t < time_period:
+        supervisor.step(timestep)
+    else:
+        ret_value = True
+
+    return ret_value
+
+
+delta = 0.001
+epsilon = 0.05
+achieved_destination = False
+
+if supervisor.getName() == 'opponent_box':
+    destination = positions[random.randint(0, len(positions) - 1)]
+
+if supervisor.getName() == 'opponent_box(1)':
+    destination = positions_1[random.randint(0, len(positions_1) - 1)]
```

Comment:
```suggestion
positions = POSITIONS_1 if supervisor.getName() == 'opponent_box' else POSITIONS_2
```
Ja bih ovako uprostio. Onda u promjenjivoj `positions` imas odgovarajuce pozicije i uvijek ih vadis sa `destination = positions[random.randint(0, len(positions) - 1)]`